packages/ciphernode/README.md

```diff
@@ -103,7 +102,32 @@ enclave start -vv
 
 if you want to remove all output aside from errors you can use the `--quiet` argument.
 
-
 ```
 enclave --quiet
 ```
+
+## Open telemetry
+
+The node can be run with open telemetry by passing in the `--otel` arg.
+
+We can run a Jaeger instance using docker:
+
+```
+docker run -e COLLECTOR_OTLP_ENABLED=true -p 16686:16686 -p 4317:4317 -p 4318:4318 jaegertracing/all-in-one:latest
```

Comment:
for me this ran v1 of jaeger, which is now deprecated. we should update this to [v2](https://www.jaegertracing.io/docs/2.5/getting-started/)

Reply:
So I updated the docker image suggestion. NOTE: jaeger still uses the old version 1 UI but it works fine from what I can tell.